examples/data-frames.dx

```diff
@@ -123,158 +128,104 @@ def addRows {n m a} (t:n=>a) (t':m=>a) : (n|m)=>a =
+  {age=age + 1, ...other}
+:p vcat students increasedAge
+
+
+curve = \grade. grade + 5
+curvedMidtermAndFinal = for i.
+  {midterm, final, ...other} = gradebook.i
+  {midterm=curve(midterm), final=curve(final), ...other}
+:p vcat gradebook curvedMidtermAndFinal
+
+
+-- moved up from below
+-- ### `dropColumns :: t1:Table * cs:Seq<ColName> -> t2:Table`
+def dropColumns {n f'} (f: Fields) (t:n=>{...f & ...f'}) : n=>{&...f'} =
+  for i.
+    {@...f=_, ...r} = t.i
+    r
+
+:p dropColumns {age: _ ? } students
+:p dropColumns {final: _ ? midterm: _} gradebook
+
+
 
 def hcat {n f f'} (t:n=>{&...f}) (t':n=>{&...f'}) : n=>{...f & ...f'} =
   for i. {...(t.i), ...(t'.i)}
 
--- > hcat(students, dropColumns(gradebook, ["name", "age"]))
--- | name    | age | favorite color | quiz1 | quiz2 | midterm | quiz3 | quiz4 | final |
--- | ------- | --- | -------------- | ----- | ----- | ------- | ----- | ----- | ----- |
--- | "Bob"   | 12  | "blue"         | 8     | 9     | 77      | 7     | 9     | 87    |
--- | "Alice" | 17  | "green"        | 6     | 8     | 88      | 8     | 7     | 85    |
--- | "Eve"   | 13  | "red"          | 7     | 9     | 84      | 8     | 8     | 77    |
--- > hcat(dropColumns(students, ["name", "age"]), gradebook)
--- | favorite color | name    | age | quiz1 | quiz2 | midterm | quiz3 | quiz4 | final |
--- | -------------- | ------- | --- | ----- | ----- | ------- | ----- | ----- | ----- |
--- | "blue"         | "Bob"   | 12  | 8     | 9     | 77      | 7     | 9     | 87    |
--- | "green"        | "Alice" | 17  | 6     | 8     | 88      | 8     | 7     | 85    |
--- | "red"          | "Eve"   | 13  | 7     | 9     | 84      | 8     | 8     | 77    |
+:p hcat students $ dropColumns {name: _ ? age: _} gradebook
+:p hcat (dropColumns {name: _ ? age: _} students) gradebook
+
 
 def values {n a} : n=>a -> n=>a = id
--- > values([
---     [row: ("name", "Alice")],
---     [row: ("name", "Bob")]])
--- | name    |
--- | ------- |
--- | "Alice" |
--- | "Bob"   |
--- > values([
---     [row: ("name", "Alice"), ("age", 12)],
---     [row: ("name", "Bob"), ("age", 13)]])
--- | name    | age |
--- | ------- | --- |
--- | "Alice" | 12  |
--- | "Bob"   | 13  |
+
+:p values [{name="Alice"}, {name="Bob"}]
+:p values [{name="Alice", age=12},
+           {name="Bob", age=13}]
+
 
 -- ### `crossJoin :: t1:Table * t2:Table -> t3:Table`
 def crossJoin {n m f f'} (t:n=>{&...f}) (t':m=>{&...f'}) : (n & m)=>{...f & ...f'} =
   for (i, j). {...(t.i), ...(t'.j)}
 
+-- Subtable is used in this example, but its definition appears to be missing.
+def subTable {n m a f'} (rows:m=>n) (f: Fields) (t: n=>{...f & ...f'}) : m=>{&...f} =
+  for i:m.
+    {@...f=v, ...} = t.(rows.i)
+    v
+
 -- > petiteJelly = subTable(jellyAnon, [0, 1], [0, 1, 2])
 -- > petiteJelly
 -- | get acne | red   | black |
 -- | -------- | ----- | ----- |
--- | true     | false | false |
--- | true     | false | true  |
+-- | =True     | =False | =False |
+-- | =True     | =False | =True  |
```

Comment:
Weird formatting? Why the `=` before booleans?

Reply:
Whoops, that's left over from a find-and-replace when I was formatting all the tables to dex tables.